lib/build.api.js

```diff
@@ -70,7 +69,7 @@ class Build {
       let project = this.aquifer.project;
 
       // Set some reasonable defaults.
-      this.options.makeFile = this.getAbsolutePath(this.options.makeFile) || project.absolutePaths.makeFile;
+      this.options.makeFile = project.absolutePaths.make;
```

Comment:
Actually, looking into it, we don't have a command option for makeFile so I'm good with this as is.